org-formation/650-identity-providers/_tasks.yaml

```diff
@@ -572,6 +572,41 @@ GithubOidcSageBionetworksSynapseProdWebMonorepoInfra:
     Account: !Ref SynapseProdAccount
     Region: us-east-1
 
+GithubOidcSageBionetworksSynapseDevInfra:
+  Type: update-stacks
+  DependsOn: GithubOidcSageBionetworks
+  Template: https://raw.githubusercontent.com/Sage-Bionetworks/aws-infra/v0.7.6/templates/IAM/github-oidc-provider.j2
+  StackName: !Sub ${resourcePrefix}-${appName}-sage-bionetworks-synapsedev-infra
```

Comment:
isn't naming it `synapsedev-infra` a bit too generic for a role that only has read access to a single bucket?  maybe come up with a better stack or role name?

Reply:
Yeah I keep going back and forth since we can support multiple repos... Renamed to specify e2e.